mvcc/backend/batch_tx.go

```diff
@@ -165,7 +166,9 @@ func (t *batchTx) commit(stop bool) {
 			atomic.StoreInt64(&t.backend.size, t.tx.Size())
 			return
 		}
+		start := time.Now()
 		err = t.tx.Commit()
+		commitDurations.Observe(time.Since(start).Seconds())
```

Comment:
Actually I should change the one in syncDuration. It keeps the nanosecond accuracy see pkg/storage/.